### ISSUE TITLE
JetBrains: Sourcegraph Cody + Code Search error when determining Git URL in version 5.2.4819

### ISSUE THREAD

**Issue by lnewson** [sourcegraph/jetbrains#130]
Hi,

Since the release of version 5.2.4819 when trying to use the `Sourcegraph` -> `Open Selection in Sourcegraph Web` option in the IntelliJ context menu I'm getting the following error. I suspect it's related to https://github.com/sourcegraph/jetbrains/commit/dd24360f9cef3a49c7b836c07e1332a2e910b544 as that's where the specific error was introduced. Everything is working correctly if I downgrade to 5.2.2301.

```
Error determining repository info: Failed to convert git clone URL to codebase name for cloneURL: org-1111111@github.com:user/repo.git"
```

Note: I've obscured the full URL, but it's a private enterprise GitHub repo if that helps.

**Comment by jdorfman** [comment]
Thanks for reporting @lnewson someone will take a look :)

**Comment by mkondratek** [comment]
Hello @lnewson 👋

It looks like I am the author of the problematic commit 😅 I am sorry to read about your inconvenience 🙇 I am going to help you 👨‍🔧 

I tried a private repo URL and version [5.2.7299](https://plugins.jetbrains.com/plugin/9682-sourcegraph-cody--code-search/versions/stable/437280) - it works fine for me.

Can you try switching to **5.2.7299**? Does the issue occur to you when you use it?

**Comment by lnewson** [comment]
Thanks for taking a look 👍 I tested the new version mentioned above and can confirm it now works. ~Looking at the changes, it looks like the fix was this commit: https://github.com/sourcegraph/cody/commit/65cbc12c00aed56c4013ef6c97736a82ac0dc294~

Edit: Sorry I misread and the above was included in 5.2.4819 so must have been something else. Either way it's working now thanks!
Edit 2: It was likely this commit that fixed things: https://github.com/sourcegraph/cody/commit/b7f9a774dde8eb6235e5ba8871f1ef1c6e70a68d
